test/KHI_growthRate/etc/picongpu/hemera-hzdr/k80_testKHI.tpl

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+# Copyright 2013-2022 Axel Huebl, Anton Helm, Richard Pausch, Rene Widera,
+#                     Marco Garten
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
```

Comment:
Please remove the tpl file from our PR. The file will be outdated soon. You can add a `README.md` into the example where you describe what the user must do to validate the results.